src/SwedbankPay.Sdk.Infrastructure/Extensions/EnumExtensions.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace SwedbankPay.Sdk.Extensions
+{
+    public static class EnumExtensions
+    {
+        public static TEnum Parse<TEnum>(String value) where TEnum : struct

```

Comment:
Make this into a proper extension method so the word `Extensions` make sense.

```suggestion
    public static class StringExtensions
    {
        public static TEnum ToEnum<TEnum>(this string value) where TEnum : struct
```